doc/whats_new.rst

```diff
@@ -69,6 +69,11 @@ Enhancements
      (`#7723 <https://github.com/scikit-learn/scikit-learn/pull/7723>`_)
      by `Mikhail Korobov`_.
 
+   - ``check_estimator`` now attempts to ensure that methods transform, predict, etc.
+     do not set attributes on the estimator.
+     (`#7553 <https://github.com/scikit-learn/scikit-learn/pull/7553>`_)
```

Comment:
okay, anyway nor my version, not this one is not clickable for me (looks like https://github.com/scikit-learn/scikit-learn/blob/master/doc/whats_new.rst#id17). Not the scope of this issue, obviously. I'll fix that!